Geotech/WekaHills_CRS_EPSG2193.ifcx

```diff
@@ -0,0 +1,51 @@
+[
+  {
+    "def": "def",
+    "name": "urn:ogc:def:crs:EPSG::2193",
+    "comment": "The name of this object is an OGC Uniform Resource Name.",
+    "type": "crs",
+    "attributes": {
+      "WKT-CRS": {
+        "ISO": "ISO 19162:2019",
```

Comment:
```suggestion
        "ISO": "ISO 19162:2019",
        "name": "urn:ogc:def:crs:EPSG::2193",
```

name would then be a property in the attributes, not sure if it needs the full urn